src/test/java/org/opentripplanner/graph_builder/module/osm/WalkableAreaBuilderTest.java

```diff
@@ -0,0 +1,134 @@
+package org.opentripplanner.graph_builder.module.osm;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import static java.util.stream.Collectors.toMap;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import java.io.File;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Consumer;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.opentripplanner.graph_builder.issue.api.DataImportIssueStore;
+import org.opentripplanner.graph_builder.services.osm.CustomNamer;
+import org.opentripplanner.openstreetmap.OpenStreetMapProvider;
+import org.opentripplanner.openstreetmap.model.OSMLevel;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.street.model.edge.AreaEdge;
+import org.opentripplanner.transit.model.framework.Deduplicator;
+
+public class WalkableAreaBuilderTest {
```

Comment:
This is a good test and the beginning of something new - thanks!

I've taken the liberty to remove a bit of duplication by using a parameterized test. It's not really a requirement to getting this merged as it comes down to personal preferences, but if you like it, you can also cherry-pick this commit: https://github.com/leonardehrenfried/OpenTripPlanner/commit/6eed5823b4faab950a89e78a1cf310909912a98e